prepublish/common/exports.js

```diff
@@ -49,6 +49,11 @@ module.exports = env => {
             // and will enable us to work with symlinked packages during development.
             symlinks: false
         },
+        resolvePlugin: {
```

Comment:
Isn't that property called `resolveLoader`?

Reply:
:+1:

Reply:
Forcepushed,
I won't merge these before we meet at the office ;)